webui/react/src/pages/TrialDetails/useTrialMetrics.ts

```diff
@@ -13,37 +13,58 @@ import handleError from 'utils/error';
 import { metricToKey } from 'utils/metric';
 
 type MetricName = string;
-
 export interface TrialMetrics {
   data: Record<MetricName, Serie>;
   metrics: Metric[];
 }
-
-const summarizedMetricToSeries = (summ: MetricContainer): Serie => {
-  const rawData: [number, number][] = [];
```

Comment:
**nit:** rename to rawBatches since that is more clear as to what this data is representing.